### DISCUSSION TITLE
Is the correct label for the previous route path?

### DISCUSSION THREAD

**Discussion by HiddenUndefined** [nuxt/devtools#422] [Q&A] [answered]
In my case is a previous routing path, but in devtools is name «next».

![Screenshot 2023-09-18 at 00 15 18](https://github.com/nuxt/devtools/assets/39486979/4563d43d-b2ac-4b89-ace8-45b6473f2119)

Maybe I'm missing something, but it seems like a small typo to me.

**Comment by arashsheyda** [comment]
@HiddenUndefined I think it's referring to next item on the list based on ASC

**Reply by HiddenUndefined** [reply]
Another case :)
<img width="661" alt="Screenshot 2023-09-18 at 02 34 21" src="https://github.com/nuxt/devtools/assets/39486979/cea759a8-589a-426f-871a-abf71b6ec045">

**Reply by arashsheyda** [reply, marked as answer]
the value of input is for finding a match route (`next` route).

as you can see in the video, after navigating from a page(`/`) to another page(`/about`), the previous rotue(`/`) stays as the input's value and that's why it says `next`.

https://github.com/nuxt/devtools/assets/38922203/7d077586-e059-41e3-ba4e-7fa332fa1ba4

**Reply by HiddenUndefined** [reply]
Oh yes, I didn't notice that the input field records the previous routing value, after your comment I went into devtools repo and [your are right](https://github.com/nuxt/devtools/blob/530244b2d4793477e926d41e3beaa8c629d168d3/packages/devtools/client/pages/modules/pages.vue#L49))
